docs/accessibility.rst

```diff
@@ -0,0 +1,182 @@
+Accessibility Testing
+======================
+
+The bok-choy framework includes the ability to test web pages for accessibility
+using `Google Accessibility Developer Tools`_.
+
+The audit rules that are checked and optionally the scope of the audit
+within the webpage itself are defined on each Page Object's definition.
+
+The general methodology for enabling accessibility auditing is:
```

Comment:
I just noticed that these bullet points actually outline the sections of the rest of the page.  Can these be linked to the corresponding sections?